docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1352,6 +1352,11 @@ Query Parameters:
 -   **memswap** - Total memory (memory + swap), `-1` to disable swap.
 -   **cpushares** - CPU shares (relative weight).
 -   **cpusetcpus** - CPUs in which to allow execution (e.g., `0-3`, `0,1`).
+-   **buildargs** – JSON map of string pairs for build-time variables. Users can
```

Comment:
The use of `environment` could be confusing, given the interaction with Environment vars, or won't that be a problem?